e2e/pages/administration/system-maintenance/system-maintenance.po.js

```diff
@@ -24,6 +24,10 @@ class SystemMaintenancePage {
   async canNavigate(target) {
     return (await $(this.elements.navigationButton(target))).isClickable();
   }
+
+  async editStandards(identifier) {
```

Comment:
I love this genericizing action, and we need to take it one step farther, as not all the things in system-maintenance are standards. Maybe:

```suggestion
  async editItem(identifier) {
```